Tests/helper.py

```diff
@@ -210,7 +210,7 @@ def skipKnownBadTest(self, msg=None, platform=None,
             self.skipTest(msg or "Known Bad Test")
 
     def tempfile(self, template):
-        assert template[:5] in ("temp.", "temp_")
+        self.assertIn(template[:5], ("temp.", "temp_"))
```

Comment:
I'm not sure about this one, it's more of an immediate test coding error, not something that should cause a test suite failure.

Reply:
Though, I'm struggling to determine why that's actually the case. It's not like we actually care about it anymore.

Reply:
Okay, I've removed it from the commit.